### PR TITLE
[chore] v0.3.0

### PR DESCRIPTION
**Pull request by blefaudeux**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## What does this PR do?
Bump up the version number to get to a new release. We can do another one with the other PRs still up in the air, not a big issue

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: fairscale/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # LICENSE file in the root directory of this source tree.
 
 # Please update the doc version in docs/source/conf.py as well.
-__version__ = "0.1.7"
+__version__ = "0.1.8"
```
Comment (min-xu-ai) [inline fairscale/__init__.py:7]: can you please update docs/source/conf.py as well?

**Review by min-xu-ai — APPROVED** [review]
Thanks!

File: fairscale/__init__.py (hunk 1 above)
Comment (min-xu-ai) [reply fairscale/__init__.py:7]: perhaps version 0.3 to signify all 3 zeros are there?

Comment (blefaudeux) [reply fairscale/__init__.py:7]: ahah, as you prefer, I think it goes beyond zero or else, these concepts are not completely new anyway. Doing that for the tongue in cheek reference

**Comment by blefaudeux** [comment]
pip package tested and deployed :)
Screen Shot 2021-02-22 at 10 59 22 PM

**Comment by min-xu-ai** [comment]
super duper berry awesome!

**Comment by myleott** [comment]
Should we also upload the pre-built wheels to pypi? You can download them from here and then upload them via twine as usual: https://github.com/facebookresearch/fairscale/actions/runs/591717171

**Pull request merged**